Makefile

```diff
@@ -38,6 +38,7 @@ install: all
 	mkdir -p $(DESTDIR)$(bindir)/
 	$(INSTALL) -pt $(DESTDIR)$(bindir)/ $(EXECUTABLES)
 	$(INSTALL) -Dp -m 644 zcfan.service $(DESTDIR)$(prefix)/lib/systemd/system/zcfan.service
+	$(INSTALL) -Dp -m 644 zcfan.1 $(DESTDIR)$(prefix)/share/man/man1/zcfan.1
```

Comment:
this should be fine. It's probably better to define `datadir` (based on prefix) and `mandir` (based on datadir), see https://www.gnu.org/prep/standards/html_node/Directory-Variables.html (it actually recommends `datarootdir` but I don't actually see that use for anything in practice).

Not *that* relevant until we have a ./configure script, so either way it's fine.